lib/index.js

```diff
@@ -4,25 +4,78 @@ var createPattern = function (path) {
   return {pattern: path, included: true, served: true, watched: false}
 }
 
-var initMocha = function (files, mochaConfig) {
+var initMocha = function (files, config) {
   var mochaPath = path.dirname(require.resolve('mocha'))
   files.unshift(createPattern(path.join(__dirname, 'adapter.js')))
 
-  if (mochaConfig && mochaConfig.require && mochaConfig.require.map) {
+  config = config || {}
+  config.client = config.client || {}
+  var mochaConfig = config.client.mocha = getMochaOpts(config.client.mocha || {})
+
+  if (mochaConfig.require && mochaConfig.require.map) {
     mochaConfig.require.map(function (requirePath) {
       return files.unshift(createPattern(requirePath))
     })
   }
 
   files.unshift(createPattern(path.join(mochaPath, 'mocha.js')))
 
-  if (mochaConfig && mochaConfig.reporter) {
+  if (mochaConfig.reporter) {
     files.unshift(createPattern(path.join(mochaPath, 'mocha.css')))
   }
 }
 
-initMocha.$inject = ['config.files', 'config.client.mocha']
+initMocha.$inject = ['config.files', 'config']
 
 module.exports = {
   'framework:mocha': ['factory', initMocha]
 }
+
+function getMochaOpts (mochaConfig) {
+  var optsPath = typeof mochaConfig.opts === 'string' ? mochaConfig.opts : 'test/mocha.opts'
+
+  if (!mochaConfig.opts) {
+    return mochaConfig
+  }
+
+  delete mochaConfig.opts
+
+  var fs = require('fs')
+  if (!fs.existsSync(optsPath)) {
+    return mochaConfig
+  }
+
+  var extend = require('util')._extend
+  return extend(normalizeOpts(require('minimist')(fs.readFileSync(optsPath, 'utf8')
+    .replace(/\\\s/g, '%20')
+    .split(/\s/)
+    .filter(Boolean)
+    .map(function (value) {
+      return value.replace(/%20/g, ' ')
+    }))), mochaConfig)
+
+  function normalizeOpts (opts) {
+    opts = [
+      'require',
+
```

Comment:
@dignifiedquire This line separte options which will pass to mocha in browser from options which use only on the server (like require) and not acceptable in browser